export-api-def/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var helpers = require('../lib/helpers');
+var apigenerator = require('loopback-api-definition');
+
+module.exports = yeoman.generators.Base.extend({
+
+  constructor: function() {
+    yeoman.generators.Base.apply(this, arguments);
+    var format = this.options.json;
+    this.format = format ? 'json' : 'yaml';
+  },
+
+  help: function() {
+    return helpers.customHelp(this);
+  },
+
+  generate: function() {
+    process.stdout.write(apigenerator.getApiDef(require(process.cwd()), {format:this.format}));
```

Comment:
There is so much being done on this single line. Would you mind splitting it into several lines, one step per line?